content/en/docs/installing-on-linux/introduction/intro.md

```diff
@@ -97,13 +72,7 @@ With KubeKey, you can scale the number of nodes to meet higher resource needs af
 
 ### Remove Nodes
 
-You need to drain a node before you remove. For more information, see Remove Nodes.
-
-### Add New Storage Classes
-
-KubeKey allows you to set a new storage class after the installation. You can set different storage classes for KubeSphere itself and your workloads.
-
-For more information, see Add New Storage Classes.
+You need to drain a node before you remove. For more information, see [Remove Nodes](../../../installing-on-linux/cluster-operation/remove-nodes/).
```

Comment:
You need to drain a node before you remove.
->
You need to drain a node before you remove it.

Reply:
done